src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ImportProcess.java

```diff
@@ -137,9 +137,21 @@ else if (coverage != null)
         		storeInfo = catalog.getCoverageStoreByName(ws.getName(), store);
         	}
             if (storeInfo == null) {
-                throw new ProcessException("Could not find store " + store + " in workspace "
-                        + workspace);
-                // TODO: support store creation
+                // mirroring "features != null" below
```

Comment:
Usual feedback, if you touch the file update the header with the latest year.

So this file should now say:

````
/* (c) 2014-2015 Open Source Geospatial Foundation - all rights reserved
 * (c) 2001 - 2013 OpenPlans
 * This code is licensed under the GPL 2.0 license, available at the root
 * application directory.
 */
````